app/views/admin/bikes/_bike.html.haml

```diff
@@ -1,32 +1,40 @@
+
 %li
-  - if bike.thumb_path.present?
-    %img{src: bike.thumb_path, alt: bike.title }
-  %h3
-    = link_to bike.title_string, bike.list_link_url(target)
-  %table.table-list
-    %tbody
-      %tr
-        %td
-          Serial number
-        %td
-          = bike.serial_number
-      %tr
-        %td
-          Colors
-        %td
-          = bike.frame_colors.to_sentence
-      - if bike.stolen and bike.current_stolen_record.present?
-        %tr
-          %td
-            Stolen at
-          %td
-            %span.convertTime.preciseTime
-              = l bike.current_stolen_record.date_stolen, format: :convert_time
-        %tr
-          %td
-            From
-          %td
-            = bike.short_address(bike.current_stolen_record)
+  .row.align-items-end
+    .col-md-2
+      #admin-bike-box
```

Comment:
There should only be one ID per page. Also, in general, you shouldn't apply styles to IDs unless you really need to